src/main/java/org/folio/circulation/domain/UpdateItem.java

```diff
@@ -208,9 +209,13 @@ private ItemStatus itemStatusOnLoanUpdate(
     Loan loan,
     RequestQueue requestQueue) {
 
-    return loan.isClosed()
-      ? itemStatusOnCheckIn(requestQueue)
-      : CHECKED_OUT;
+    if(loan.isClosed()) {
+      return itemStatusOnCheckIn(requestQueue);
+    }
+    if(Objects.equals(loan.getItem().getStatus(), ItemStatus.DECLARED_LOST)) {
```

Comment:
I think this should either be an `else if` or there be a blank line separating the if statements

Reply:
Sure, done.